xml/dm_registers.xml

```diff
@@ -287,6 +292,9 @@
       The position of the window is determined by \Rhawindowsel. I.e. bit 0
       refers to hart $\Rhawindowsel * 32$, while bit 31 refers to hart
       $\Rhawindowsel * 32 + 31$.
+
+      Bits that are outside the hart array mask for this DM are not writable,
```

Comment:
do we need to explicitly say that the hart array mask register may not be any larger than the number of harts in the system? Otherwise it's a little unclear what it means to be "outside the hart array mask". Is it a legal implementation to allow, say, the bottom 13 harts to be selected with hart array, but not the rest of the 200 harts on my system?

Reply:
The hart array mask register explicitly may be larger than the number of harts in the system. That allows people to have a DM module that supports 64 harts that they can just plug into a system that has <65 harts.

I guess it would be legal to not have all your harts be selectable, but that doesn't seem like something anybody would want to do. Maybe it's best not to allow it.

Reply:
I've clarified this.

Reply:
Maybe you should just remove this sentence again, since you clarified above that some bits of the register can be tied to 0. I still think "outside the hart array mask" is unclear. Maybe you could change this sentence to say:

"Since some bits in the hart array mask register may be constant 0, not all bits of this register are always R/W, depending on the current value of hawindowsel".

Reply:
Done (and then edited just a tiny bit).
This is ready for review again. :-)